torch_xla/csrc/xla_graph_executor.h

```diff
@@ -163,8 +163,14 @@ class XLAGraphExecutor : public torch::lazy::LazyGraphExecutor {
   };
 
   using ComputationCache =
+      runtime::util::AbstractCache<torch::lazy::hash_t, CachedComputation,
+                                   torch::lazy::HashReducer>;
+  using MemoryCache =
       runtime::util::Cache<torch::lazy::hash_t, CachedComputation,
```

Comment:
I am confuse about the difference between `ComputationCache ` and `MemoryCache `

Reply:
`ComputationCache` is used for the umbrella return type, and `MemoryCache` and `PersistentCache` are subtypes of `ComputationCache`

Reply:
The types were split out in https://github.com/pytorch/xla/pull/5800